libc/string_cpy.c

```diff
@@ -0,0 +1,1451 @@
+/*
+ * Phoenix-RTOS
+ *
+ * POSIX.1-2017 standard library functions tests
+ * HEADER:
+ *    - string.h
+ * TESTED:
+ *    - memcpy()
+ *    - memccpy()
+ *    - strncpy()
+ *    - stpncpy()
+ *    - strcpy()
+ *    - stpcpy()
+ *    - strlcpy()
+ *    - strlcat()
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Mateusz Niewiadomski, Damian Modzelewski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <unity_fixture.h>
+
+/* defines lengths of buffers holding strings*/
+#define MAX_STR_LEN   24
+#define MAX_ASCII_SET 256
+#define BIG_NUMB      1024
+
+/* defines for string and mem copy */
+#define TEST_STR1 "Lorem ipsum dolor"
+#define TEST_STR2 "Maecenas id commodo"
+
+/* defines for string_strlcpy */
+#define STR_SRC  "abcd"
+#define STR_DEST "xxxx"
+
+/* defines for string_strlcat */
+#define STR_SRC1        "abc"
+#define STR_SRC2        "defgh"
+#define STR_PLACEHOLDER "klmnopqrstu"
+
+
+static char longStr[BIG_NUMB];
+
+
+static int big_string(void)
```

Comment:
There is a common convention to indicate what the function is doing in a name.

```suggestion
static int create_big_string(void)
```